### PR TITLE
Remove registry.clear() method. Update tests to use the safe-tag-name he...

### PR DESCRIPTION
**Pull request by treshugart**
...lper.

### CODE REVIEW

**Comment by treshugart** [comment]
Fixes #140.

File: test/unit/registration.js [hunk 1]
```diff
@@ -51,34 +53,24 @@ describe('Returning a constructor', function () {
...
 
-    Div.prototype.func2 = function () {};
-
-    var div = new Div();
-
-    expect(div.func1).to.be.a('function');
-    expect(div.func2).to.be.a('function');
-  });
-
-  it('should allow the overwriting of the prototype', function () {
```
Comment (nicolasv) [inline test/unit/registration.js]: What is the new behaviour of attempting what this test did? Perhaps it's worth keeping the test around to ensure the new behaviour remains.

**Comment by nicolasv** [comment]
:+1:

File: test/unit/registration.js (hunk 1 above)
Comment (treshugart) [inline test/unit/registration.js]: We could write a test to make sure the prototype cannot be overwritten, but I'm not sure if we need to go that far. It's just not a feature anymore.

**Pull request merged**